### PR TITLE
Organisation custom footer

### PR DESCRIPTION
**Pull request by ansmonjol**
Closes https://github.com/getlago/lago-front/issues/203

This MR adds the possibility to add and edit a custom footer, that will be displayed in the generated customer invoice.

Also to download a customer invoice, with the edited organisation's footer

image
image

### CODE REVIEW

File: src/components/customers/CustomerInvoicesList.tsx [hunk 1]
```diff
@@ -47,6 +58,27 @@ const mapStatus = (type?: InvoiceStatusTypeEnum | undefined) => {
...
+        if (myWindow?.location?.href) {
+          myWindow.location.href = fileUrl
+          return myWindow?.focus()
+        } else {
+          myWindow?.close()
+        }
+      }
+
+      addToast({
+        severity: 'danger',
```
[the remark this answers is not in the document]
Comment (ansmonjol) [reply src/components/customers/CustomerInvoicesList.tsx:77]: Yes, according to designs if any error occurs it should display this red toast 

image

[the remark this answers is not in the document]
Comment (MorganeLecurieux) [reply src/components/customers/CustomerInvoicesList.tsx:77]: Sorry not sure to understand... here you add a toast "onCompleted"... So... There's no error happening ? 🤔

File: src/pages/settings/VatRate.tsx [hunk 2]
```diff
@@ -1,43 +1,119 @@
...
-    : 0
+  const [isTextTruncated, setIsTesxtTruncated] = useState(true)
+  const editVATDialogRef = useRef<EditOrganizationVatRateDialogRef>(null)
+  const editInvoiceTemplateDialogRef = useRef<EditOrganizationInvoiceTemplateDialogRef>(null)
+  const { data, error, loading } = useGetOrganizationInvoiceAndTaxInformationsQuery()
+  const organization = (data?.currentUser?.organizations || [])[0]
+  const vatRate = organization?.vatRate || 0
+  const invoiceFooter = organization?.invoiceFooter || ''
+
+  const renderInvoiceFooter = () => {
```
Comment (MorganeLecurieux) [inline src/pages/settings/VatRate.tsx:55]: I wonder if this couldn't be a component ?

Comment (MorganeLecurieux) [reply src/pages/settings/VatRate.tsx:55]: Looks like something we could reuse

Comment (ansmonjol) [reply src/pages/settings/VatRate.tsx:55]: on it!

Comment (ansmonjol) [reply src/pages/settings/VatRate.tsx]: Done [in the last commit](https://github.com/getlago/lago-front/pull/209/commits/61f4b4132cee689a53abc7abd3cd84d4f2b831ab), let me know what you think!

**Review by MorganeLecurieux — APPROVED** [review]

**Pull request merged**